frontend_lib/src/component/Tags/NewTagForm.styl

```diff
@@ -0,0 +1,92 @@
+@import '../../css/Variable.styl'
+
+.taglist__form
+  padding 15px
+  flex-direction column
+  height 100%
+  width 100%
+  &__title
+    font-size 20px
```

Comment:
Can you use the font size defined at Variable.styl?

Reply:
done